customers.sql

```diff
@@ -0,0 +1,34 @@
+with customers as (
+    SELECT 
+        * 
+    FROM 
+        analytics-engineers-club.coffee_shop.customers
+),
+
+orders as (
+    SELECT 
+        *
+    FROM 
+        analytics-engineers-club.coffee_shop.orders
+),
+
+customers_orders as (
```

Comment:
**suggestion to improve query performance**: rather than joining all of the columns from `customers` and `orders`, we could use this CTE to aggregate `first_order_at` and `number_of_orders` on `customer_id` in the `orders` table, and then join these results to `customers` in the last part of the query.